components/automate-ui/README.md

```diff
@@ -74,34 +74,23 @@ error TS6200: Definitions of the following identifiers conflict with those in an
 There are very few reports of others experiencing this problem and no resolution presents itself.
 Reference: https://stackoverflow.com/a/57592510
 
-**Package @types/node: ~12.12.29**
+### Package @types/node: ~12.12.29
 
 Reason: The major version should match our node version, which is 12.13.1, so I have constrained it to the highest version less than that.
 That is a guess on my part, could not really confirm what version one should be running, through these references:
- - https://www.npmjs.com/package/@types/node
- - https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library
 
-**Package diff2html: ^2.12.2**
```

Comment:
I patched breaking changes from this package and unlocked the version. However, need someone to actually test this in the UI; I could not quite tell where it shows up.

Reply:
@Shadae and/or @scottopherson: this file has your fingerprints, so when you have a chance, please check (or tell me how to)>

Reply:
@msorens the only place I see `diff2html` being used is in `app-delta-viewer` which is only used by `app-resource-item`. Everything seems to look fine on the client-runs node details page which has the only usage of `app-resource-item`:

![Screen Shot 2020-05-05 at 12 05 05 PM](https://user-images.githubusercontent.com/479121/81102942-92c44e80-8ecd-11ea-86fd-ff9545980937.png)

(though the padding within the diff rows seems a bit much imo)